### ISSUE TITLE
Update privacy notice on pre-release pages

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#5792]
Update the privacy notice on [https://www.mozilla.org/en-US/firefox/channel/desktop/](https://www.mozilla.org/en-US/firefox/channel/desktop/).

Get PR ready to go and send to l10n but do not merge until June 26.

**Comment by stephaniehobson** [comment]
Discussions about what we're updating it to are ongoing.

The client needs 2 weeks to communicate the change, hence the June 26 launch date.

**Comment by stephaniehobson** [comment]
This is up on demo now: https://bedrock-demo-shobson.oregon-b.moz.works/en-US/firefox/channel/desktop/

@ejregithub Can you please check in with the client to make sure what's there is okay?

**Comment by stephaniehobson** [comment]
This issue is a duplicate of #5746

**Comment by ejregithub** [comment]
I pinged me Mika in slack DM, requesting she QA and provide comments/approval here (she has a github login and therefore can do so).

**Comment by mikashah** [comment]
The anchor in the Privacy notice hyperlinks ("Learn what data is shared") isn't currently working.  Will that be fixed later?

**Comment by stephaniehobson** [comment]
@mikashah It sends me to https://bedrock-demo-shobson.oregon-b.moz.works/en-US/privacy/firefox/#telemetry Is that not right / not what it's doing for you?
